### PR TITLE
Ruttenberg/hotspot to gateway

### PR DESCRIPTION
**Pull request by ruttenberg**
Update field names and types for packet up.

### CODE REVIEW

File: src/grpc/hpr_packet_up.erl [hunk 1]
```diff
@@ -40,15 +40,16 @@ payload(Packet) ->
...
-    Packet#packet_router_packet_up_v1_pb.signal_strength.
+-spec rssi(Packet :: packet()) -> non_neg_integer() | undefined.
+rssi(Packet) ->
+    Packet#packet_router_packet_up_v1_pb.rssi.
 
--spec frequency(Packet :: packet()) -> float().
-frequency(Packet) ->
-    Packet#packet_router_packet_up_v1_pb.frequency.
+-spec frequency_mhz(Packet :: packet()) ->
+    float() | integer() | infinity | '-infinity' | nan | undefined.
```
Comment (dpezely) [inline src/grpc/hpr_packet_up.erl:48]: Frequency is a float, per conversations with Michael (on vacation).
We can omit `integer` here.
```suggestion
    float() | infinity | '-infinity' | nan | undefined.
```

**Review by dpezely — APPROVED** [review]
Thanks for accommodating those pesky name changes and modifications to the draft protobufs!

**Review by hwahby — APPROVED** [review]

File: src/grpc/hpr_packet_up.erl (hunk 1 above)
Comment (ruttenberg) [reply src/grpc/hpr_packet_up.erl:48]: @dpezely 
The record definition has this:
         frequency_mhz = 0.0    :: float() | integer() | infinity | '-infinity' | nan | undefined, % = 5, optional

I thought it best to match the record definition.
If you think it is better to take out 'integer()' anyway, I am happy to do so.

cc/ @hwahby

Comment (dpezely) [reply src/grpc/hpr_packet_up.erl:48]: Ah, okay.  Keep it consistent with the record definition as you have it.  Thanks for clarifying.

**Pull request merged**